app/src/main/AndroidManifest.xml

```diff
@@ -30,6 +29,7 @@
             android:name=".SettingsActivity"
             android:label="@string/title_activity_settings"></activity>
         <activity
+            android:exported="true"
```

Comment:
This is needed for apps using intent-filters that target android version 12 or higher. See information here:
https://developer.android.com/guide/topics/manifest/activity-element#exported
tldr; allows other apps to start this activity